app/src/main/res/menu/feeditem_options.xml

```diff
@@ -72,28 +72,6 @@
         android:id="@+id/share_item"
         android:menuCategory="container"
         android:title="@string/share_label">
```

Comment:
I would use `share_label_with_ellipses` here

Reply:
yup, I forgot update this . Thanks